src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTask.java

```diff
@@ -0,0 +1,64 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.basic.TopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrFilmDeserializer;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrFilmPartDeserializer;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+
+public class WdrFilmDetailTask extends AbstractDocumentTask<Film, TopicUrlDTO>  {
+
+  private static final Logger LOG = LogManager.getLogger(WdrFilmDetailTask.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)